internal/pkg/cli/task_run.go

```diff
@@ -26,37 +36,57 @@ var (
 	taskRunEnvPromptHelp = fmt.Sprintf("Task will be deployed to the selected environment. "+
 		"Select %s to run the task in your default VPC instead of any existing environment.", color.Emphasize(config.EnvNameNone))
 	taskRunGroupNamePromptHelp = "The group name of the task. Tasks with the same group name share the same set of resources, including CloudFormation stack, CloudWatch log group, task definition and ECR repository."
+
+	fmtTaskFamilyName = "copilot-%s"
+	fmtRepoName       = "copilot-%s"
+	fmtImageURL       = "%s:%s"
+)
+
+const (
+	defaultImageTag       = "copilot"
+	defaultDockerfilePath = "./Dockerfile"
```

Comment:
I don't think the `"./"` will work on windows 😃, maybe we can just keep it as `"Dockerfile"`?